nncf/common/pruning/symbolic_mask.py

```diff
@@ -109,3 +111,9 @@ def elementwise_mask_propagation(cls, input_masks: List[SymbolicMask]) -> Symbol
                 return AmbiguousSymbolicMask(producers)
 
         return SymbolicMask(input_masks[0].shape[0], producers)
+
+    @classmethod
+    def split(cls, tensor: SymbolicMask, chunks: int, axis: int) -> List[SymbolicMask]:
+        producer = tensor.mask_producers
+        tensor_shape = math.floor(tensor.shape[0]/chunks)
+        return [SymbolicMask(tensor_shape, producer) for _ in range(chunks)]
```

Comment:
```suggestion
        producers = tensor.mask_producers
        tensor_shape = np.floor(tensor.shape[0]/chunks)
        return [SymbolicMask(tensor_shape, producers) for _ in range(chunks)]
```

Reply:
In this part, we need to use math instead of numpy. When we use numpy, the bool result in line 117 of mask_propagation.py could be np.bool_.